roles/openshift_docker/tasks/main.yml

```diff
@@ -23,6 +23,6 @@
   with_items:
   - role: docker
     local_facts:
-      openshift_image_tag: "{{ l_image_tag }}"
-      openshift_version: "{{ l_image_tag if l_image_tag is defined else '' | oo_image_tag_to_rpm_version }}"
-  when: openshift.common.is_containerized is defined and openshift.common.is_containerized | bool
+      openshift_image_tag: "{{ l_image_tag | default(None) }}"
+      openshift_version: "{{ l_image_tag | oo_image_tag_to_rpm_version }}"
+  when: openshift.common.is_containerized | bool
```

Comment:
Same question here.  If `openshift.common.is_containerized is defined` isn't need then we should remove it.  I just don't see how it's related to the bug in question.